go/chat/convsource.go

```diff
@@ -140,12 +143,36 @@ func (s *HybridConversationSource) Pull(ctx context.Context, convID chat1.Conver
 					MsgID:          localData.Messages[0].GetMessageID(),
 				})
 				if err != nil {
-					return chat1.ThreadView{}, nil, err
+					return chat1.ThreadView{}, nil, nil, err
 				}
 				rl = append(rl, res.RateLimit)
 			}
 
-			return localData, rl, nil
+			var breaks []keybase1.TLFUserBreak
```

Comment:
But we can't store the proof breaks in local storage since they might sit there for a long time and by the time they get used the identify result might be different.

I thought we cache the identify for 6 hours or something like that. Is it not true?